tests/main-tests.scm

```diff
@@ -1,18 +1,21 @@
-(define-module (tests main)
-	#:use-module (srfi srfi-64)
-	#:use-module (ice-9 futures)
-	#:use-module (opencog)
-	#:use-module (opencog bioscience)
-	#:use-module (annotation main)
-	#:use-module (annotation functions)
-	#:use-module (annotation util)
+#! /usr/bin/env guile
+!#
+
+(use-modules
+	(srfi srfi-64)
+	(ice-9 futures)
+	(opencog)
+	(opencog bioscience)
+	(annotation main)
+	(annotation functions)
+	(annotation util)
 )
 
 (test-begin "main")
 
-(primitive-load-path "opencog_deps")
 ;; Load test atomspace
-(primitive-load-path "tests/sample_dataset.scm")
+(primitive-load "../tests/sample_dataset.scm")
```

Comment:
This results in test failure. Reverting back to the previous version (`tests/sample_dataset.scm`) fixes the test failure.